jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/executor/SingleThreadPolicyRunner.java

```diff
@@ -104,16 +107,18 @@ public List<PolicyResult> runPolicies(List<Map<String, String>> resources, Map<S
                                                                                                                      // allow
                                                                                                                      // modify
                                                                                                                      // input
-                    } catch (Exception e) {
+                    }
+                    catch (Exception e) {
```

Comment:
revert

Reply:
i just formattted.

Reply:
I'm not sure that this was done by reformating. Ususally it would be converted into `} catch (Exception e) {` in single line.

Reply:
reverted.